src/client.rs

```diff
@@ -130,13 +136,17 @@ impl ClientBuilder {
                     endpoint,
                     request_timeout_secs,
                     pool_idle_timeout_secs,
+                    http2_keep_alive_interval_secs,
+                    http2_keep_alive_while_idle,
                 },
             signer,
             connector,
         } = self;
         let http_client = HttpClient::builder(TokioExecutor::new())
             .pool_idle_timeout(pool_idle_timeout_secs.map(Duration::from_secs))
             .http2_only(true)
+            .http2_keep_alive_interval(http2_keep_alive_interval_secs.map(Duration::from_secs))
+            .http2_keep_alive_while_idle(http2_keep_alive_while_idle)
```

Comment:
Found a panic during testing. Please include the timer:

```suggestion
            .http2_keep_alive_while_idle(http2_keep_alive_while_idle)
            .timer(TokioTimer::new())
```

Reply:
I tested with this change and the connection resets have stopped. Thanks again for this change